build.sbt

```diff
@@ -13,6 +13,10 @@ scalaVersionsByJvm in ThisBuild := {
   )
 }
 
+lazy val root = project.in(file("."))
+  .aggregate(`scala-parser-combinatorsJS`, `scala-parser-combinatorsJVM`)
+  .settings(publish := {}, publishLocal := {})
```

Comment:
The scala-module-plugin defines a list of settings `disablePublishing` that you can use if you like: https://github.com/scala/sbt-scala-modules/blob/v1.0.8/src/main/scala/ScalaModulePlugin.scala#L65-L74